packages/react-instantsearch-hooks/src/lib/useSearchResults.ts

```diff
@@ -21,10 +21,15 @@ export function useSearchResults(): SearchResultsApi {
 
   useEffect(() => {
     function handleRender() {
-      setSearchResults({
-        results: searchIndex.getResults()!, // Results can't be `null` after the first render.
-        scopedResults: searchIndex.getScopedResults(),
-      });
+      const results = searchIndex.getResults();
+
+      // results can be null when first search is stalled, in this case we skip the update
```

Comment:
1. Can you add a permalink to the relevant code from the Helper? This will be useful to understand this portion of code later on when stumbling upon it.

Reply:
I think it has to do with `instantsearch.js` and how the index widget works, I added a link to that piece of code which made me understand the issue : https://github.com/algolia/instantsearch.js/blob/20996c7a159988c58e00ff24d2d2dc98af8b980f/src/widgets/index/index.ts#L652-L657